lisp/bazel-build.el

```diff
@@ -24,38 +24,52 @@
 
 ;;; Code:
 
+(defgroup bazel-build 'nil
+  "Package for editing, building, and running code using Bazel."
+  :link '(url-link "https://bazel.build")
+  :link '(url-link "https://github.com/bazelbuild/emacs-bazel-mode")
+  :group 'languages)
+
+(defcustom bazel-build-bazel-command "bazel"
+  "Command that should be used to invoke Bazel."
+  :type 'string
+  :group 'bazel-build)
+
 (require 'bazel-util)
 
 (defun bazel-build (target)
   "Build a Bazel TARGET."
-  (interactive (list (bazel-build--read-target "bazel build ")))
+  (interactive (list (bazel-build--read-target "build")))
   (bazel-build--run-bazel-command "build" target))
 
 (defun bazel-run (target)
   "Build and run a Bazel TARGET."
-  (interactive (list (bazel-build--read-target "bazel run ")))
+  (interactive (list (bazel-build--read-target "run")))
   (bazel-build--run-bazel-command "run" target))
 
 (defun bazel-test (target)
   "Build and run a Bazel test TARGET."
-  (interactive (list (bazel-build--read-target "bazel test ")))
+  (interactive (list (bazel-build--read-target "test")))
   (bazel-build--run-bazel-command "test" target))
 
 (defun bazel-build--run-bazel-command (command target)
   "Run Bazel tool with given COMMAND, e.g. build or run, on the given TARGET."
   (compile
-   (mapconcat #'shell-quote-argument (list "bazel" command target) " ")))
+   (mapconcat #'shell-quote-argument
+              (list bazel-build-bazel-command command target) " ")))
 
-(defun bazel-build--read-target (prompt)
-  "Read a Bazel build target from the minibuffer.  PROMPT is a read-only prompt."
+(defun bazel-build--read-target (command)
+  "Read a Bazel build target from the minibuffer. COMMAND is a read-only
```

Comment:
Nit: Emacs style is to have a single sentence as the first line of a docstring. (M-x checkdoc helps find such issues.)

Reply:
TIL, thanks! Fixed.